include/vcpkg/base/message-data.inc.h

```diff
@@ -1387,7 +1387,7 @@ DECLARE_MESSAGE(HashFileFailureToRead,
 DECLARE_MESSAGE(HashPortManyFiles,
                 (msg::package_name, msg::count),
                 "",
-                "The {package_name} contains {count} files. Hashing these contents may take a long time when "
+                "The port '{package_name}' contains {count} files. Hashing these contents may take a long time when "
```

Comment:
```suggestion
                "{package_name} contains {count} files. Hashing these contents may take a long time when "
```

Dropping "the <kind> '<name>'" was an intentional change but leaving 'the' there was not 😅

Thanks for the report!

Reply:
166ecc10ee54d8cb5f12aba237281d2648fff09e. Thanks!